test/e2e/framework/workflows_utils.go

```diff
@@ -28,19 +28,24 @@ func IsWorkflowFailedDueDeadline(w *wapi.Workflow) bool {
 }
 
 // BuildAndSetClients builds and initilize workflow and kube client
-func BuildAndSetClients() (versioned.Interface, *clientset.Clientset) {
+func BuildAndSetClients() (versioned.Interface, versioned.Interface, *clientset.Clientset) {
 	f, err := NewFramework()
 	Ω(err).ShouldNot(HaveOccurred())
 	Ω(f).ShouldNot(BeNil())
 
 	kubeClient, err := f.kubeClient()
 	Ω(err).ShouldNot(HaveOccurred())
 	Ω(kubeClient).ShouldNot(BeNil())
-	Logf("Check wether Workflow resource is registered...")
+
 	workflowClient, err := f.workflowClient()
 	Ω(err).ShouldNot(HaveOccurred())
 	Ω(workflowClient).ShouldNot(BeNil())
-	return workflowClient, kubeClient
+
+	cronWorkflowClient, err := f.workflowClient()
```

Comment:
should be `f.cronWorkflowClient()` but I don't think we need a workflowClient and a cronWorkflowClient since they return the same interface.

Reply:
I agree. Going to change this. Thanks

Reply:
Still the wrong client

Reply:
done